build.gradle

```diff
@@ -660,6 +660,7 @@ project(':connect:runtime') {
 
   dependencies {
     compile project(':connect:api')
+    compile project(':connect:json')
```

Comment:
I don't think we want `connect-runtime` to depend on `connect-json`. Why should the runtime depend on a specific converter?